app/policies/collection_policy.rb

```diff
@@ -16,6 +17,11 @@ def update?
     administrator? || manages_collection?(record)
   end
 
+  sig { returns(T::Boolean) }
+  def update_first_draft?
```

Comment:
I don't understand why we need a separate policy for this.  Isn't what `update?` is doing sufficient?

Reply:
this policy ensures you don't end up on the page that is only designed for editing first drafts if you aren't actually editing a first draft -- there aren't any links that will take you there, but you could craft one in the browser -- this just guards against that, so its not critical

Reply:
I just removed the policy changes -- the collection policy and redirects should be sufficient.